server/mailet/mock-smtp-server/pom.xml

```diff
@@ -41,6 +41,10 @@
             <groupId>${james.groupId}</groupId>
             <artifactId>james-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-spark</artifactId>
```

Comment:
I don't really like that we expand the use of spark-java in the codebase, it doesn't bring us enough value in comparison of doing things by hand.
Could we use JettyHttpServer instead?

Reply:
Funny, I proposed during the grooming to rely on Play.

The general consensus against me was "let's choose the fastest implementation possible" & "we already all know spark" & "this is testing code" which basically everyone seemed to agree with.

So I respected the overall consensus and proposed that work...

Reply:
Please refer to https://github.com/linagora/james-project/pull/2625 for a Jetty version of this work!

Reply:
I would argue the JettyHttpServer is even simpler. Thank you for handling that.